src/main/java/org/mifos/processor/bulk/HealthCheck.java

```diff
@@ -61,12 +62,14 @@ public void configure() {
                     exchange.setProperty(BATCH_ID, batchId);
 
                     // TODO: How to get sender information? Hard coded in Channel connector?
-                    byte[] csvFile = fileTransferService.downloadFile(fileName, bucketName);
+                    InputStream csvFile = fileTransferService.downloadFile(fileName, bucketName);
```

Comment:
rename the variable to indicate that its a stream
Suggestion:  csvFileInputStream

Reply:
The method should be renamed as well.  Because right now the operation you are doing is not a full download

Reply:
Renamed